### DISCUSSION TITLE
How to use components in this repository in the platformio project

### DISCUSSION THREAD

**Discussion by John-Chan** [espressif/esp-iot-solution#443] [General]
thx

**Comment by leeebo** [comment]
@John-Chan I am not very familiar with PlatformIO, but it appears they have their own [library manager](https://docs.platformio.org/en/latest/librarymanager/index.html#librarymanager), which is not directly compatible with [ESP-IDF's component manager](https://docs.espressif.com/projects/esp-idf/en/stable/esp32/api-guides/tools/idf-component-manager.html).

My suggestion is to manually download the components from the [ESP registry](https://components.espressif.com/) and then add them as local components in PlatformIO.

**Reply by John-Chan** [reply]
Thanks, I will try your suggestion
